### PR TITLE
fix incorrect  _embedding_bag meta

### PR DESCRIPTION
**Pull request by bdhirsh**
Fixes https://github.com/pytorch/pytorch/issues/92286. See the issue for diagnosis.

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #92549

### CODE REVIEW

**Review by albanD — APPROVED** [review]
Good catch

File: test/test_fake_tensor.py [hunk 1]
```diff
@@ -743,6 +743,26 @@ def test_like_ops(self):
...
+            False,
+            2, # mode = max
+        ]
+
+        ref_out = torch.ops.aten._embedding_bag(*args)
+        with FakeTensorMode() as m:
+            meta_args = [m.from_tensor(a) if isinstance(a, torch.Tensor) else a for a in args]
+            meta_out = torch.ops.aten._embedding_bag(*meta_args)
+
+        for ref_o, meta_o in zip(ref_out, meta_out):
```
Comment (eellison) [inline test/test_fake_tensor.py:762]: nit: `self.assertTrue(len(ref_out), meta_out))`

**Review by eellison — APPROVED** [review]
nice!

**Pull request closed without merging**